Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -16,6 +16,27 @@ import UIKit
 import ARKit
 import ArcGIS
 
+extension AGSDeviceOrientation {
+    /// Allows creation of an `AGSDeviceOrientation` from a `UIDeviceOrientation`.
+    ///
+    /// - Parameter statusBarOrientation: The `UIDeviceOrientation` to create the `AGSDeviceOrientation` from.
+    init?(statusBarOrientation: UIDeviceOrientation) {
```

Comment:
This initializer is failable, but it doesn't actually ever fail (it never returns `nil`). Perhaps it either shouldn't be failable or the `default` case should be removed?

Reply:
The initializer can fail because the code in each case is fail-able:

`self.init(rawValue: AGSDeviceOrientation.landscapeRight.rawValue)`  //<<== this can fail

I thought the same thing (that it wouldn't be fail-able), but it's a compile error to not make it optional.

Reply:
My apologies. I didn't look closely enough and assumed you were force unwrapping the initialized value.

I would suggest not using `init(rawValue:)` to make the initializer non-failable:

```swift
switch statusBarOrientation {
case .landscapeLeft:
    self = .landscapeRight
case .landscapeRight:
    self = .landscapeLeft
case .portrait:
    self = .portrait
case .portraitUpsideDown:
    self = .reversePortrait
default:
    self = .landscapeLeft
}
```

I commented that last time around, but must have hit the wrong button.

Reply:
That much cleaner, thanks!